R/colocalization_heatmap.R

```diff
@@ -11,22 +11,31 @@ globalVariables(
 #' test computed with \code{\link{RunDCA}}.
 #'
 #' @param data A \code{tbl_df} object generated with \code{\link{RunDCA}}
+#' @param marker1_col The name of the column with the first marker
+#' @param marker2_col The name of the column with the second marker
+#' @param value_col The name of the column with the value to plot, such as the estimate of a differential analysis test.
 #' @param colors A character vector with colors
 #' @param return_plot_data Return data formatted for plotting
 #' instead of drawing the heatmap
+#' @param symmetrise Set to \code{TRUE} if only the lower or upper triangle of marker combinations exist in \code{data},
+#'                   each row will then be mirrored to fill the missing triangle of the heatmap.
 #' @param ... Parameters passed to \code{pheatmap}
 #'
 #' @return A heatmap
 #'
 #' @export
 #'
 ColocalizationHeatmap <- function (
```

Comment:
This could be a good time to remove this part:
```
# Declarations used in package check
globalVariables(
  names = c('estimate'),
  package = 'pixelatorR',
  add = TRUE
)
```
And add `'estimate'` in the aaa.R script instead.

Reply:
Check! ✅